docs/installing.rst

```diff
@@ -148,13 +148,30 @@ The following virtualization providers are suggested for the respective operatin
     # Set libvirtd to start automatically on system boot
     $ sudo systemctl enable libvirtd
 
------------------------------------------------------
-3. Install the vagrant-service-manager Vagrant plugin
------------------------------------------------------
+----------------------------------
+3. Install additional dependencies
+----------------------------------
+
+For some operating systems, you might need to install additional dependencies before you install the Vagrant plugins.
+
+* Fedora 24
+
+  Run the following commands to install the additional dependencies::
+
+  $ sudo dnf install "Development Tools"
+  $ sudo dnf install rpm-build zlib-devel ruby-devel
```

Comment:
Found another dependency for vagrant plugins in Fedora 24 i.e. `sudo dnf install gcc-c++` . It provides g++ compiler and without it landrush did not get installed.